src/components/SettingsManager/SharedSessionManager/SharedSessionManager.Room.tsx

```
@@ -47,52 +114,52 @@ export default function Room({ room }: RoomProps) {
+        >
+          {!qrCode ? "Generate" : "Hide"} QR Code
+        </Button>
+      </Box>
+      {qrCode && (
+        <div>
+          <Typography variant="body2" sx={{ marginBottom: 2 }}>
+            Scan this QR code from another device to synchronize its state with
+            this device.
+          </Typography>
+          <AutoScaler style={{ width: 150, height: 150, marginBottom: "1rem" }}>
+            <div dangerouslySetInnerHTML={{ __html: qrCode }} />
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)